config/settings.yml

```diff
@@ -357,6 +358,7 @@ defaults: &defaults
     lock_deployment_max_duration:  86400 # lock ttl for deployments in seconds. Its kept at 24 hrs - 60 * 60 * 24
     infrastructure: &director_infrastructure
       stemcell:
+        alias: service_fabrik_stemcell
```

Comment:
many things in the setting.yml which are IAAS specific can be removed. ex. azs, disk_types, vm_types & compilaton. 
Just keep `networks` as we might need it. But rest all things IAAS specific should be removed.